packages/shared/src/components/Feed.tsx

```diff
@@ -282,6 +283,12 @@ export default function Feed<T>({
     }
   }, [selectedPost]);
 
+  const onShareClick = useCallback(
```

Comment:
Why was the callback needed here, did it have issues?

Reply:
We will use this function as a dependency way down so we need it to be a stable callback.